### ISSUE TITLE
Need to revamp how dependencies are forcefully loaded

### ISSUE THREAD

**Issue by wyldckat** [blueCFD/Core#70]
At the end of fixing Issue #7, it was found that `objdump` cannot properly retrieved DLL dependencies from the executables. On the other hand, the `ldd` utility isn't entirely reliable either, since it sometimes fails to properly check the DLL dependencies.

The other objective is to trim down considerably a full build and not having users need to do intermediate steps themselves.

Therefore, the fix that will be implemented for blueCFD-Core 2017 will be as follows:
1. `wmakeVerifyDependencies` will be merged into the code that handles versioning preparations.
2. The `wmake` stack will be prepared to use the new convention of relying on the new inclusion file that will be generated by the code in the previous step.

**Comment by wyldckat** [comment]
It looks like the error message:

```
X:\msys64\mingw64\bin\objdump.exe: failed to read symbol table from: X:/OpenFOAM-5.x/platforms/mingw_w64GccDPInt32Opt/lib/libOpenFOAM.dll
X:\msys64\mingw64\bin\objdump.exe: error message was: File truncated
```

This is mostly a warning message, not as much an error.

However, the last build time was 192 minutes, which is somewhat excessive, when compared to the old 120 minutes it used to do in the older builds. And using `wmakeVerifyDependencies` is a nuisance, so I will proceed with changing this.

**Comment by wyldckat** [comment]
Have implemented this in commit [780ccdf87b730b9c51ae9397eb0dc2331aba041c](https://github.com/blueCFD/OpenFOAM-dev/commit/780ccdf87b730b9c51ae9397eb0dc2331aba041c).

After doing a partial fix in #69, I'll give an updated time account for a full build.

**Comment by wyldckat** [comment]
OK, clean build now takes 130 minutes to run, which is a considerable improvement and recovery.

Will close this task, since it's now complete.
